Gems/ROS2/Code/Source/Lidar/LidarBase.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <Atom/RPI.Public/AuxGeom/AuxGeomDraw.h>
+#include <AzCore/Component/EntityId.h>
+#include <AzCore/Serialization/SerializeContext.h>
+#include <ROS2/Lidar/LidarRegistrarBus.h>
+#include <ROS2/Lidar/LidarSystemBus.h>
+#include <ROS2/Sensor/ROS2SensorComponent.h>
+
+#include "LidarRaycaster.h"
+#include "LidarSensorConfiguration.h"
+
+namespace ROS2
+{
+    //! Lidar Base.
+    //! Lidars (Light Detection and Ranging) emit laser light and measure it after reflection.
+    //! Lidar Base allows for simulation of Lidars.
+    class LidarBase
```

Comment:
The naming could suggest base class, which this isn't. 
Could be named LidarCore, etc.
Also please improve class documentation (first sentence) to describe the purpose of this class.